openedx/adg/admin_dashboard/webinars/forms.py

```diff
@@ -0,0 +1,155 @@
+"""
+Forms for webinars app.
+"""
+import datetime
+
+import pytz
+from django import forms
+from django.conf import settings
+from django.contrib.auth.models import User
+
+from openedx.adg.common.lib.zoom_client.client import ZoomClient
+from openedx.adg.lms.webinars.models import Webinar
+
+from .helpers import format_checkboxes_data, format_webinar_options
+
+
+class WebinarForm(forms.ModelForm):
+    """
+    Webinar Form to create/edit a webinar from admin dashboard
+    """
+
+    start_time = forms.DateTimeField(required=False)
+    panelists = forms.CharField(required=False)
+    duration = forms.DurationField(required=False)
+    webinar_options = forms.CharField(required=False)
+
+    class Meta:
+        model = Webinar
+        fields = [
+            'title',
+            'description',
+            'language',
+            'meeting_link',
+            'webinar_timezone',
+            'audio_settings',
+            'enable_host_video',
+            'enable_panelist_video',
+            'banner',
+            'invites_by_email_address',
+        ]
+
+    def __init__(self, *args, **kwargs):
+        """
+        Override to initialize the added fields
+        """
+        super(WebinarForm, self).__init__(*args, **kwargs)
+        self.fields['meeting_link'].disabled = True
+
+        self.initialize_fields()
+        self.context = self.get_context_data()
+
+    def clean_start_time(self):
+        """
+        Format user selected day/month/year/hour/min to a webinar start_time
+        """
+        day = self.data.get('start_time_day')
+        month = self.data.get('start_time_month')
+        year = self.data.get('start_time_year')
+        hour = self.data.get('start_time_hour')
+        minute = self.data.get('start_time_min')
+        timezone = self.data.get('webinar_timezone')
+        try:
+            date_time = datetime.datetime(int(year), int(month), int(day), int(hour), int(minute))
+            return pytz.timezone(timezone).localize(date_time)
+        except (ValueError, TypeError):
+            return None
```

Comment:
Shouldn't we be raising a validation error in case our datetime object is not created successfully for some reason?